src/components/views/rooms/RoomHeader.tsx

```diff
@@ -117,7 +107,7 @@ export default function RoomHeader({ room }: { room: Room }): JSX.Element {
             gap="var(--cpd-space-3x)"
             className="mx_RoomHeader light-panel"
             onClick={() => {
-                showOrHidePanel(RightPanelPhases.RoomSummary);
+                setPhase(RightPanelPhases.RoomSummary);
```

Comment:
seems like showOrHidePanel was actually a better name:

 (a) it reflects the fact that it can also hide the panel
 (b) "phase" doesn't mean much on its own. Phase of what?